### PR TITLE
[wallet] [rpc] [doc] importprivkey: hint about importmulti

### PR DESCRIPTION
**Pull request by kallewoof**
From #12701, a hint about `importmulti` inside the help for `importprivkey` seems useful.

### CODE REVIEW

**Comment by promag** [comment]
I would say "Always prefer importmulti", even for one key. IMO `importprivkey` is redundant and ideally could be deprecated.

**Comment by kallewoof** [comment]
Really? Huh, okay. I don't think that's widely known.

**Comment by sipa** [comment]
Yes, I think `importmulti` is superior in every way, except:
* Slightly less convenient to use (it requires redundant information, as you have to explicitly state what you want to import, and then give the keys/... to enable that).
* Doesn't support P2SH-P2WPKH yet.

The latter can be fixed by just adding support for that case. The former can be improved by adding 'default' cases (e.g. if you prove a WIF private key, it can permit omitting the address, ...).

File: src/wallet/rpcdump.cpp [hunk 1]
```diff
@@ -100,6 +100,7 @@ UniValue importprivkey(const JSONRPCRequest& request)
         throw std::runtime_error(
             "importprivkey \"privkey\" ( \"label\" ) ( rescan )\n"
             "\nAdds a private key (as returned by dumpprivkey) to your wallet. Requires a new wallet backup.\n"
+            "\nHint: use importmulti to import more than one private key!\n"
```
Comment (jnewbery) [inline src/wallet/rpcdump.cpp]: Style nits. Feel free to ignore:

- doesn't require a blank line between `Adds a private key...` and `Hint: use...`
- remove the `!`. `.` will do!

**Review by jnewbery — COMMENTED** [review]
I agree that @sipa's two points here: https://github.com/bitcoin/bitcoin/pull/12702#issuecomment-373571161 should be addressed, but I don't see any harm in adding this hint until those are done.

ACK 8fb733756bc6119958a9807d83bde1a4a99cac31 with a couple of style nits.

File: src/wallet/rpcdump.cpp (hunk 1 above)
Comment (kallewoof) [reply src/wallet/rpcdump.cpp]: Done!

**Comment by jnewbery** [comment]
Tested ACK 4e05687153c69bbbfe8c5281d49b76488e02d2f7 . Looks good.

**Pull request merged**